### DISCUSSION TITLE
Opentype features? Specifically: prefer slashed zero

### DISCUSSION THREAD

**Discussion by zambetti** [zed-industries/zed#6542] [Feature Requests]
Does anyone know if there is a way to select opentype font features?

The mono type face I prefer has a slashed zero, but not as a default.

It seems VSCode handles this with "editor.fontLigatures"

Using the patterns in settings.json, I suppose font features could be selected like this?

"buffer_font_features" :
{
    "slashed_zero": true
}

**Comment by jansol** [comment]
The keys in`buffer_font_features` are the names of the opentype features you want to enable/disable. You'll have to check which feature controls them in the font you are using as that appears to not be very consistent. The webpage of the font should contain a listing of features and what they do, but that failing you can drop the file onto https://fontdrop.info/ and it'll show you a listing and attempt to show the effect of each feature (it has trouble with ligatures, but glyph variants at least should be fine).

**Reply by zambetti** [reply]
Thank you jansol, this worked for me...

"buffer_font_features" :
{
"zero": true
}

Looks like this was added just a few weeks ago in 0.7.9.1, perfect!
https://docs.zed.dev/configuration/configuring-zed
